src/wallet/rpcwallet.cpp

```diff
@@ -2634,6 +2634,31 @@ UniValue fundrawtransaction(const JSONRPCRequest& request)
     return result;
 }
 
+// Calculate the size of the transaction assuming all signatures are max size
+// Use DummySignatureCreator, which inserts 72 byte signatures everywhere.
+// TODO: re-use this in CWalletTx::CreateTransaction (right now now
```

Comment:
Oh I had originally tried to write this so that CalculateMaximumSignedTxSize() would be a function in CWallet, and CreateTransaction would directly use it -- but that was harder than I expected since CreateTransaction actually uses the dummy-signed tx for something that bumpfee doesn't care about (the priority calculation I mentioned).

So this todo refers to making these code paths even more similar than they are now.